src/Examples/PiTop.Interactive.Rover/RoverRobotSetup.cs

```diff
@@ -0,0 +1,116 @@
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.DotNet.Interactive;
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.CSharp;
+
+using PiTop.Camera;
+using PiTop.MakerArchitecture.Expansion;
+using PiTop.MakerArchitecture.Expansion.Rover;
+using PiTop.MakerArchitecture.Foundation;
+
+using Pocket;
+
+using SixLabors.ImageSharp;
+
+using UnitsNet;
+
+using static Pocket.Logger;
+namespace PiTop.Interactive.Rover
+{
+    public class RoverRobotSetup
+    {
+        public static async Task SetupKernelEnvironment(CSharpKernel csharpKernel)
+        {
+            using var _ = Log.OnEnterAndExit();
+
+            var piTopExtension = new InteractiveExtension.KernelExtension();
+            await piTopExtension.OnLoadAsync(csharpKernel);
+
+            var cameraExtension = new Camera.InteractiveExtension.KernelExtension();
+            await cameraExtension.OnLoadAsync(csharpKernel);
+
+            var foundationExtension = new MakerArchitecture.Foundation.InteractiveExtension.KernelExtension();
+            await foundationExtension.OnLoadAsync(csharpKernel);
+
+            var expansionExtension = new MakerArchitecture.Expansion.InteractiveExtension.KernelExtension();
+            await expansionExtension.OnLoadAsync(csharpKernel);
+
+            PiTop4Board.Instance.UseCamera();
+            var rover = new RoverRobot(PiTop4Board.Instance.GetOrCreateExpansionPlate(), PiTop4Board.Instance.GetOrCreateCamera<StreamingCamera>(0),
+                RoverRobotConfiguration.Default);
+            var strategies = new RoverRobotStrategies();
+
+            rover.AllLightsOn();
+            rover.BlinkAllLights();
+
+            await RegisterTypes<Image>(csharpKernel);
+            await RegisterTypes<Color>(csharpKernel);
+            await RegisterTypes<Speed>(csharpKernel);
+            await RegisterTypes<PiTop4Board>(csharpKernel);
+            await RegisterTypes<StreamingCamera>(csharpKernel);
+            await RegisterTypes<FoundationPlate>(csharpKernel);
+            await RegisterTypes<ExpansionPlate>(csharpKernel);
+            await RegisterTypes<RoverRobot>(csharpKernel);
+            await csharpKernel.SetVariableAsync("roverRobot", rover);
```

Comment:
I would align these names and use `nameof(rover)` to make things a little clearer.